sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/implementation/MultipartFormDataHelperTest.java

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.openai.implementation;
+
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.util.BinaryData;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Unit tests for MultipartFormDataHelper.
```

Comment:
nit pick: linking the class in the comment
```suggestion
 * Unit tests for {@link MultipartFormDataHelper}
```